docs/uperf.md

```diff
@@ -189,6 +192,55 @@ To enable Multus in Ripsaw, here is the relevant config.
       ...
 
 ```
+### Scale
+Scale in this context refers to the ability to enumerate UPERF 
+client-server pairs during test in a control fashion using the following knobs.
+
+`colocate: true` will place each client and server pod pair on the same node.
+
+`density_range` to specify the range of client-server pairs that the test will iterate.
+
+`node_range` to specify the range of nodes that the test will iterate.
+
+`step_size` to specify the incrementing method.
+
+Here is one scale example:
+
+```
+      ...
+      pin: false
+      colocate: false
+      density_range: [1,10]
+      node_range: [1,128]
+      step_size: log2
+      ...
+```
+Note, the `scale` mode is mutual exlusive to `pin` mode with the `pin` mode has higher precedence.
+In other words, if `pin:true` the test will deploy pods on `pin_server` and `pin_client` nodes
+and ignore `colocate`, `node_range`, and the number of pairs to deploy is specified by the
+ `density_range.high` value.
+
+In the above sample, the `scale` mode will be activated since `pin: false`. In the first phase, the 
+pod instantion phase, the system gathers node inventory and may reduce the `node_range.high` value 
+to match the number of worker node available in the cluster.
+
+According to `node_range: [1,128]`, and `density_range:[1,10]`, the system will instantiate 10 pairs on 
```

Comment:
Update "will instantiate 10 pairs" by "will instantiate up to 10 pairs"

Reply:
It will instantiate 10 pairs not "up to".  For nodes we may reduce if the number of nodes in the cluster is less than CR. However, for pod, it will do as density_range specifies.